### ISSUE TITLE
require all columns not only the ones that must not be missing for omopTables

### ISSUE THREAD

**Issue by catalamarti** [darwin-eu-dev/omopgenerics#519]
https://github.com/darwin-eu-dev/omopgenerics/blob/7cbd021667650f22fdbad450157322e4032e49b1/R/classOmopTable.R#L42 
TURN THIS TO FALSE

**Comment by catalamarti** [comment]
this would currently break:
- omock @ilovemane 
- IncidencePrevalence @edward-burn 
- OmopSketch @martaalcalde 
- CohortSymmetry @xihang-chen 
- CodelistGenerator @edward-burn 
- PatientProfiles @catalamarti 

Would be nice if we can make sure that mocks contain all the required columns in the new versions of the package I will open a separate issue in each one of the packages

**Comment by edward-burn** [comment]
I don't think it is worth breaking six packages for this. At most I suppose it could be added as a warning initially
